dashboard/README.md

```diff
@@ -74,6 +74,22 @@ pytest --cov
 ```
 
 ## How it works
+#### `aurora_map.py`
+#### `azimuth_plot.py`
+#### `hold.py`
+#### `Homepage.py`
+#### `load_dashboard.py`
+#### `nasa_pipeline.py`
```

Comment:
Could have more informative descriptions of each file in the future

Reply:
Good to specifically mention each script needed for the dashboard, in the future it'd be nice to have each file name  hyperlink to each file for user / reader convenience.

Reply:
Good use of markdowns for consistent smaller formatting.